src/instantsearch/instantsearch.ts

```diff
@@ -283,7 +234,7 @@ export class NgAisInstantSearch implements AfterViewInit, OnInit, OnDestroy {
 
   public instantSearchInstance: InstantSearchInstance;
 
-  constructor(@Inject(PLATFORM_ID) private platformId: Object) {}
+  constructor(@Inject(PLATFORM_ID) public platformId: Object) {}
```

Comment:
this is used in a test, so needs to be public

Reply:
I think you mean in base-widget.ts?

https://github.com/algolia/angular-instantsearch/pull/699/files/79e3446af931bdaffa69ef4f0451221916755961#diff-dd8934600aab087352ec0aec0ef3b4baR38

Reply:
yes, in baseWidget, not a test. Good catch :)